src/v5_hal/firmware/src/DataNodes/ADIAnalogOutNode.cpp

```diff
@@ -0,0 +1,23 @@
+#include "DataNodes/ADIAnalogOutNode.h"
+
```

Comment:
Everything looks good, this is just a test to show you how comments from code reviews integrate with VSCode. Pretend I wanted you to change this constructor to something else (I don't), we will respond to the comment from inside of VSCode

Reply:
ok